### PR TITLE
introduce font encoding

### PR DESCRIPTION
**Pull request by PeterPablo**
as discussed in issue #587

### CODE REVIEW

**Comment by PeterPablo** [comment]
I also added input encoding to handle umlauts and accents.

**Comment by PeterPablo** [comment]
This change of course influences all hashes... I have to get back to this later.

**Comment by PeterPablo** [comment]
As soon as you think this is ready, I will update the hash maps. It probably doesn't make sense to do this right now.

**Comment by PeterPablo** [comment]
Any opinions? Since I/we need to regenerate all hashes when applying this PR, what do you think of introducing indentation to the m2t output (e.g. one tab `\t` in front of everything following `\begin{axis}`)?
Currently I do this by hand to increase legibility. Of course we can go with this PR as is and not mix up matters. It's just an idea...

**Comment by okomarov** [comment]
Adding a tab means we have less characters for long table when compiling or am I wrong? If yes, then I would keep un-indented.

**Comment by PeterPablo** [comment]
You mean concerning the memory constraint of latex? I am uncertain, whether latex trims whitespace before compilation.

**Comment by egeerardyn** [comment]
Please do nog change the format any further in this PR. It's a different feature, so do that another time if we really want to do that. Updating the hashes is no problem, but I dislike making the other change in a haphazardly.

**Comment by PeterPablo** [comment]
Fine by me. Actually I think `m2t` behaved differently in an earlier release but let's continue discussion in #597.

**Comment by PeterPablo** [comment]
Travis is happy. Please give this a spin and merge, if satisfied.

**Comment by egeerardyn** [comment]
I cannot fully reproduce your hashes here:

R2014a:
```
The following tests are known to be unreliable. They, however, do not cause the build to fail.

           ACID(  8) peaks_contour            : hash 4fdcc78af7120bb5d4b447490647d75b != (30922fbe7410b5a4151874b610b9a736)
           ACID( 10) peaks_contourf           : hash 2485f8996b224e78f055543ac37fc8e1 != (b1913e13f40beef74da65d6a463fef3a)
           ACID( 17) colorbarLogplot          : hash 25d5dc87bffcc5504ee97f800695d06e != (0f6344a9e3fc0b9cb38ea3918602b8ef)
           ACID( 18) legendplot               : hash 47900312e18c39a133f9b6342d77060c != (dafe7a9566de01abdee039ec8af9cd29)
           ACID( 29) polarplot                : OK
           ACID( 30) roseplot                 : OK
           ACID( 31) compassplot              : OK
           ACID( 43) rlocusPlot               : OK
           ACID( 45) besselImage              : hash 72cf026d11959a8a67add7191f136b4a != (2be5300f1f8dc4baf1b00931159408c5)
           ACID( 47) zplanePlot1              : OK
           ACID( 48) zplanePlot2              : hash c652dda17bd9ef8068272631bca980f9 != (559b627b4e8c86681e8ceee990d9a639)
           ACID( 49) freqResponsePlot         : OK
           ACID( 56) spherePlot               : OK
           ACID( 57) surfPlot                 : hash e68770779077b85b87e5751bbecf8afa != (b7d444599c2ca0345caa86ac5feb05bf)
           ACID( 58) surfPlot2                : hash 1389a6887a5e7853a0748af54328c7e8 != (79e4fb104ed1e63f8518b135f993c978)
           ACID( 60) meshPlot                 : OK
           ACID( 62) spectro                  : OK
           ACID( 63) mixedBarLine             : OK
           ACID( 67) texrandom                : OK
           ACID( 69) latexmath2               : OK
           ACID( 70) parameterCurve3d         : OK
           ACID( 71) parameterSurf            : hash 14b760693935cf31e831b01a8ecebd1d != (cd9fc11fdde062ad437ea87b95b4ce2c)
           ACID( 75) hist3d                   : OK
           ACID( 81) pColorPlot               : OK
           ACID( 82) hgTransformPlot          : OK
           ACID( 88) annotationSubplots       : hash 574808c022462e00d49e4e67fe1c2d7a != (20c66d9564da246aefec3e646cf183a2)
           ACID( 89) annotationText           : hash bec1dc85c70b268d004bb76c27176bd7 != (657dd6d52f7b482ef0da830c0f7c9d5c)
           ACID( 90) annotationTextUnits      : hash 48257ff7f3939bec236801bdc2f401af != (12ad28a8902e0555017e697f405f0287)
           ACID( 95) colorbarLabelTitle       : hash c0eac65b9f2426faaffb3929d1e1da89 != (3670447cbf70b47e2e8104ce5ebdf63e)
           ACID( 96) textAlignment            : hash 89540a20345d812467708bda17c9d130 != (035983dfbd573c6ed628285466ef663a)

 14 of  30 tests failed. :-( 


Only the following tests determine the build outcome:
           ACID(  1) multiline_labels         : OK
           ACID(  2) plain_cos                : OK
           ACID(  3) sine_with_markers        : OK
           ACID(  4) markerSizes              : OK
           ACID(  5) markerSizes2             : OK
           ACID(  6) sine_with_annotation     : OK
           ACID(  7) linesWithOutliers        : OK
           ACID(  9) contourPenny             : OK
           ACID( 11) many_random_points       : OK
           ACID( 12) double_colorbar          : hash 9f8df1f04215873f5eb8d50252213b78 != (d5e1cdd066750729cc5cad0d22abaae1)
           ACID( 13) randomWithLines          : OK
           ACID( 14) double_axes              : OK
           ACID( 15) double_axes2             : OK
           ACID( 16) logplot                  : hash f5053634ccd961fc6d95c48df7f95be9 != (e14feacb3479ac5a8b4dd8294b096132)
           ACID( 19) legendplotBoxoff         : OK
           ACID( 20) moreLegends              : OK
           ACID( 21) zoom                     : OK
           ACID( 22) quiveroverlap            : OK
           ACID( 23) quiverplot               : OK
           ACID( 24) quiver3plot              : hash 4cd7eb850e401213cc238dd9ba127686 != (c92a354f492e093710abcf6687e0c1bd)
           ACID( 25) logicalImage             : OK
           ACID( 26) imagescplot              : OK
           ACID( 27) imagescplot2             : OK
           ACID( 28) stairsplot               : OK
           ACID( 32) stemplot                 : OK
           ACID( 33) stemplot2                : OK
           ACID( 34) bars                     : hash 4ec9811b3ba6e24bc10bddd655a56f40 != (7afb3f0415c45b489d97fdfd047692b1)
           ACID( 35) xAxisReversed            : OK
           ACID( 36) errorBars                : OK
           ACID( 37) errorBars2               : OK
           ACID( 38) subplot2x2b              : OK
           ACID( 39) manualAlignment          : OK
           ACID( 40) subplotCustom            : hash bf227900a56905bf7c39d653314d8667 != (a40207079e1fd8116df7c027a21570b4)
           ACID( 41) legendsubplots           : OK
           ACID( 42) bodeplots                : OK
           ACID( 44) mandrillImage            : OK
           ACID( 46) clownImage               : OK
           ACID( 50) axesLocation             : OK
           ACID( 51) axesColors               : OK
           ACID( 52) multipleAxes             : OK
           ACID( 53) scatterPlotRandom        : OK
           ACID( 54) scatterPlot              : OK
           ACID( 55) scatter3Plot             : OK
           ACID( 59) superkohle               : OK
           ACID( 61) ylabels                  : OK
           ACID( 64) decayingharmonic         : OK
           ACID( 65) texcolor                 : OK
           ACID( 66) textext                  : OK
           ACID( 68) latexInterpreter         : OK
           ACID( 72) fill3plot                : OK
           ACID( 73) rectanglePlot            : OK
           ACID( 74) herrorbarPlot            : OK
           ACID( 76) myBoxplot                : hash a57e07a28a39cfd6dad154f2e714c3de != (5381722f000fae74074598ad0d602c51)
           ACID( 77) areaPlot                 : OK
           ACID( 78) customLegend             : hash a5b8f598d5f811aeb16aa8716585b32a != (41b082fc4d06edfe00730c37b3b5156e)
           ACID( 79) pixelLegend              : OK
           ACID( 80) croppedImage             : OK
           ACID( 83) scatterPlotMarkers       : OK
           ACID( 84) multiplePatches          : OK
           ACID( 85) logbaseline              : OK
           ACID( 86) alphaImage               : OK
           ACID( 87) annotationAll            : hash 5e724d943ab1166c7440ffe4a0072482 != (cdab7295cba22a536c067f82e8780ae5)
           ACID( 91) imageOrientation_PNG     : OK
           ACID( 92) imageOrientation_inline  : OK
           ACID( 93) texInterpreter           : OK
           ACID( 94) stackedBarsWithOther     : OK
           ACID( 97) overlappingPlots         : OK
           ACID( 98) histogramPlot            : SKIPPED
           ACID( 99) alphaTest                : OK

  8 of  69 tests failed. :-(
```

In R2014b, I have much better results:
```
The following tests are known to be unreliable. They, however, do not cause the build to fail.

           ACID(  1) multiline_labels         : hash dd24b7ba70c454a3efad69fea49c046d != (694b79095ebec59b6c0fc3be9994c834)
           ACID(  6) sine_with_annotation     : hash 23fd0965ff3d3d5d5b036def35749fe8 != (a798aea05716f8d6e9fe1a83b4b2315d)
           ACID(  9) contourPenny             : OK
           ACID( 10) peaks_contourf           : hash 7976036f384e89a1e36234ccf0108887 != (25613ef0545cf90a2ad752af0e098bff)
           ACID( 12) double_colorbar          : hash eddb534807b0d947298c294ca8a69ba6 != (53bd9e05fe4bc1927a3034aa7081ee7d)
           ACID( 16) logplot                  : hash b53044e62cc3f959a53574d878ea157f != (6780b1273775f673b1c8eabc0b3cb2e8)
           ACID( 17) colorbarLogplot          : SKIPPED
           ACID( 18) legendplot               : hash 6113cde41ade9815c7e442fa0262758e != (9b7ced19b13439f6b18a46f8731fbd43)
           ACID( 20) moreLegends              : OK
           ACID( 24) quiver3plot              : hash 82cadac0d4ac57e0583ea1f2e2938e23 != (467191f50fb514bcefea7d9c575e9dbf)
           ACID( 34) bars                     : hash 22b541cb700b7b103b98d26d4a2d79fb != (5c4ab393176da0579b631d6b868105b8)
           ACID( 38) subplot2x2b              : hash ae1dad5878f79954215069f74ac3f6d2 != (cf296f94a540b38b889cf0929709bf8e)
           ACID( 39) manualAlignment          : OK
           ACID( 40) subplotCustom            : hash 31f99c143f502abd41f0502c752707b2 != (384eb5f1c6310ffceacccd1fb3b92354)
           ACID( 45) besselImage              : hash 69eb16d1e98ec24643934adb885bd7ac != (9800927ec29da23ac9e6431a53c7ab23)
           ACID( 48) zplanePlot2              : hash f1e685e3d0f46b542a70d8f6c23397d6 != (c561f7f3d080d6721fe73241cf3483f2)
           ACID( 49) freqResponsePlot         : OK
           ACID( 52) multipleAxes             : OK
           ACID( 57) surfPlot                 : hash 65b3a78c6245765c6bdbb6add370a8af != (7fbdc3e3bd94cb47ecc376588f723ade)
           ACID( 58) surfPlot2                : hash d055396e8c3ac5d804cbf9bf13f56a96 != (cfcaf8b7cf464f83fd500b37c2dc7713)
           ACID( 61) ylabels                  : OK
           ACID( 63) mixedBarLine             : OK
           ACID( 67) texrandom                : OK
           ACID( 68) latexInterpreter         : hash 7f9957c8c30d46654053ff4b72a07801 != (d9b7e757aac4248f83061316a494a744)
           ACID( 73) rectanglePlot            : hash 75b4cedeb9671ebffe51d578a36bf607 != (3c5ce72fe99e09caea456fcb4c281cff)
           ACID( 88) annotationSubplots       : OK
           ACID( 90) annotationTextUnits      : OK
           ACID( 91) imageOrientation_PNG     : OK
           ACID( 92) imageOrientation_inline  : OK
           ACID( 94) stackedBarsWithOther     : hash 8453772c298fce90966c51055cded833 != (92b2d4d3c6c5b0559373b2ff0bb92ff0)
           ACID( 95) colorbarLabelTitle       : OK
           ACID( 96) textAlignment            : hash 1ea1552e69585481bc15a2978100aaf2 != (ed5814a306462c4367f45400479bbd23)
           ACID( 97) overlappingPlots         : hash 67646edacef32aa958516e7a5ae92963 != (ec4195e1e1ff498abf3ce62864f11b04)

 19 of  33 tests failed. :-( 


Only the following tests determine the build outcome:
           ACID(  2) plain_cos                : OK
           ACID(  3) sine_with_markers        : OK
           ACID(  4) markerSizes              : OK
           ACID(  5) markerSizes2             : OK
           ACID(  7) linesWithOutliers        : OK
           ACID(  8) peaks_contour            : OK
           ACID( 11) many_random_points       : OK
           ACID( 13) randomWithLines          : OK
           ACID( 14) double_axes              : OK
           ACID( 15) double_axes2             : OK
           ACID( 19) legendplotBoxoff         : OK
           ACID( 21) zoom                     : OK
           ACID( 22) quiveroverlap            : OK
           ACID( 23) quiverplot               : OK
           ACID( 25) logicalImage             : OK
           ACID( 26) imagescplot              : OK
           ACID( 27) imagescplot2             : OK
           ACID( 28) stairsplot               : OK
           ACID( 29) polarplot                : OK
           ACID( 30) roseplot                 : OK
           ACID( 31) compassplot              : OK
           ACID( 32) stemplot                 : OK
           ACID( 33) stemplot2                : OK
           ACID( 35) xAxisReversed            : OK
           ACID( 36) errorBars                : OK
           ACID( 37) errorBars2               : OK
           ACID( 41) legendsubplots           : OK
           ACID( 42) bodeplots                : OK
           ACID( 43) rlocusPlot               : OK
           ACID( 44) mandrillImage            : OK
           ACID( 46) clownImage               : OK
           ACID( 47) zplanePlot1              : OK
           ACID( 50) axesLocation             : OK
           ACID( 51) axesColors               : OK
           ACID( 53) scatterPlotRandom        : OK
           ACID( 54) scatterPlot              : OK
           ACID( 55) scatter3Plot             : OK
           ACID( 56) spherePlot               : OK
           ACID( 59) superkohle               : OK
           ACID( 60) meshPlot                 : OK
           ACID( 62) spectro                  : OK
           ACID( 64) decayingharmonic         : OK
           ACID( 65) texcolor                 : OK
           ACID( 66) textext                  : OK
           ACID( 69) latexmath2               : OK
           ACID( 70) parameterCurve3d         : OK
           ACID( 71) parameterSurf            : OK
           ACID( 72) fill3plot                : OK
           ACID( 74) herrorbarPlot            : OK
           ACID( 75) hist3d                   : OK
           ACID( 76) myBoxplot                : OK
           ACID( 77) areaPlot                 : OK
           ACID( 78) customLegend             : OK
           ACID( 79) pixelLegend              : OK
           ACID( 80) croppedImage             : OK
           ACID( 81) pColorPlot               : OK
           ACID( 82) hgTransformPlot          : OK
           ACID( 83) scatterPlotMarkers       : OK
           ACID( 84) multiplePatches          : OK
           ACID( 85) logbaseline              : OK
           ACID( 86) alphaImage               : OK
           ACID( 87) annotationAll            : OK
           ACID( 89) annotationText           : OK
           ACID( 93) texInterpreter           : OK
           ACID( 98) histogramPlot            : OK
           ACID( 99) alphaTest                : OK

All tests were successful. :-) 
```

I.e. in R2014a, the tests `[12, 16, 24, 34, 40, 76, 78, 87]` are either unreliable or the hashes are wrong.
I've placed my R2014a output in:
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test12-converted.tex
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test16-converted.tex
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test24-converted.tex
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test34-converted.tex
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test40-converted.tex
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test76-converted.tex
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test78-converted.tex
* https://dl.dropboxusercontent.com/u/452005/m2t/PR590/test87-converted.tex

@PeterPablo Can you check this against yours or copy your output such that I can have a look?

**Comment by PeterPablo** [comment]
```
--- yours/test12-converted.tex
+++ mine/test12-converted.tex
@@ -30,7 +30,7 @@
 \begin{tikzpicture}
 
 \begin{axis}[%
-width=0.2966\figurewidth,
+width=0.29651\figurewidth,
 height=\figureheight,
 at={(0\figurewidth,0\figureheight)},
 scale only axis,
@@ -1527,7 +1527,7 @@
 \end{axis}
 
 \begin{axis}[%
-width=0.2966\figurewidth,
+width=0.29651\figurewidth,
 height=\figureheight,
 at={(0.540296\figurewidth,0\figureheight)},
 scale only axis,
```
So this one seems unreliable due to precision. This should hopefully vanish after #604.

The next two tex files I got the same output as you. I will shorty update the hashes. May be I messed something up before. I think I did this under Linux, but am at the moment on Mac.

**Comment by PeterPablo** [comment]
:-( I have to rebase/merge now, otherwise Travis won't execute...

**Comment by PeterPablo** [comment]
@egeerardyn, please have another look. How should I proceed with reducing the number of commits? Rebasing against develop and squashing commits?

**Comment by okomarov** [comment]
I reckon you can rebase it on top of develop, nobody has branched from it and I do not think you can selectively drop commits (http://stackoverflow.com/questions/8939808/git-remove-unwanted-commits-from-a-pull-request). It can create more problems than advantages.

**Comment by egeerardyn** [comment]
@PeterPablo : One should never rebase public code* . Don't worry about the many commits: just merge `develop` back into this branch, i.e. in your command line (assuming your `develop` branch is up-to-date with `matlab2tikz/develop`):
```bash
git checkout improve_standalone
git merge develop
```

Many commits is a good thing: it allows one to track what you did, how you did it. Rebasing (and squashing) is something you should only do before you start with a PR, e.g. if you have done some things that are half-working (e.g. I sometimes do it when a commit introduced an error, I notice that only after a few commits, so I make a fixup commit (that I squash against the offensing commit) to hide my mistakes from the rest of you). When the commits are in the open, rebasing is not helpful, since that makes it harder to track the discussions here and the corresponding commits. It also clutters the repositories with objects that cannot be reached in a logical fashion.

*: unless you are really, really sure that nobody has ever watched that code, which you don't.

**Comment by PeterPablo** [comment]
Thank you both for elaborating. I saw developers being asked to squash/rebase their commits of pull requests in larger projects (e.g. OwnCloud, CyanogenMod) in order to remove those intermediate commits that don't add any value and would result in an uncompileable project state, if e.g. the following commit would be reverted.
I totally understand that rebasing in `develop` or `master` is a no go, though have the impression that it is fine inside a PR, if consent is reached and intermediate (non-working) states should be removed.

Such intermediate and non-working commits are present here and I wondered whether those should be squashed in a way that there is for example only a single commit that updates hashes.

**Comment by egeerardyn** [comment]
Rebasing a long-living branch should indeed be not done (except in very exceptional cases).
For our PRs it's indeed somewhat less of an issue, although it depends on if multiple people could be working on the same branch, how many comments and commits there have been on the PR (i.e. when it's commented on a lot, rebasing might make it hard to follow long discussions). I sometimes fetch a branch from someone else and make changes on top to test it. That becomes a mess once rebases come into play.

In this branch, I'd say that the extra clarity one gets from squashing is quite limited.

**Comment by PeterPablo** [comment]
Ok :smiley: :+1: I try to follow your suggestions.
---
As you see I updated the hashes. It seems something went wrong on my side previously / or my linux machine behaves differently.

**Comment by egeerardyn** [comment]
Only 1 failing test in R2014b (4 in R2015a, but I will ignore those for the time being):
```
The following tests are known to be unreliable. They, however, do not cause the build to fail.

           ACID(  1) multiline_labels         : OK
...
           ACID( 16) logplot                  : OK
           ACID( 17) colorbarLogplot          : SKIPPED
           ACID( 34) bars                     : hash 22b541cb700b7b103b98d26d4a2d79fb != (a23bc34ad5753f4c26c8830bd4dd90c3)
           ACID( 39) manualAlignment          : hash 8f3496f0248c332a05c90288de69d831 != (4998fd5fe4e1a040db3efe0798285730)
           ACID( 48) zplanePlot2              : hash f1e685e3d0f46b542a70d8f6c23397d6 != (dea6dd06d6f4eff417569707a9c3693f)
           ACID( 63) mixedBarLine             : hash 142ad2cfdef55a99046a279f46d8fee7 != (277810c06a95e95c39be98d142207fb3)
           ACID( 94) stackedBarsWithOther     : hash 8453772c298fce90966c51055cded833 != (af92ecb46c9f99663e4719bde03ce457)

  5 of  33 tests failed. :-( 


Only the following tests determine the build outcome:
           ...
           ACID( 36) errorBars                : hash bbfefbee478b11b426c97ec1176b8bed != (94d66a27df67621e2c75e6041df46411)
           ...

  1 of  66 tests failed. :-( 
```

[ACID(36) in R2014b](https://gist.github.com/egeerardyn/fd3d82e4d1d42b03479f)

*currently running R2014a tests, will update upon completion*

**Comment by PeterPablo** [comment]
Diff for ACID(36) on R2014b:
```
--- yours/converted/test36-converted.tex
+++ mine/converted/test36-converted.tex
@@ -34,7 +34,7 @@
        3	       3	0.0682695	0.418709\\
        4	       4	0.308668	0.0138193\\
        5	       5	0.425776	0.219683\\
-       6	       6	9.05151e-12	0.237666\\
+       6	       6	1.3745e-11	0.237666\\
        7	       7	0.425776	0.219683\\
        8	       8	0.308668	0.0138193\\
        9	       9	0.0682695	0.418709\\
```
Sorry, I can't make sense out of this.

Will update soon...

**Comment by okomarov** [comment]
Math library might be the reason (?). 

I am gonna reference http://undocumentedmatlab.com/blog/math-libraries-version-info-upgrade for general info and link it up to #524 (generate debugging info) and #552 (resolve hash mismatches)

**Comment by PeterPablo** [comment]
@egeerardyn, as pointed out above the failing/unreliable tests are not caused by this PR. We need to tacke #414 and #552 to resolve those. By now we know, where things tend do go wrong. I will try to remove the dependency of `svd()`. `m2t` shouldn't struggle because of how we generate data for testing the functionality.

**Comment by egeerardyn** [comment]
@PeterPablo: I agree with your diagnosis of the problem. If you mark those tests as unreliable in the corresponding environments (and add a little comment at each occurrence with a link back to here), I think we can merge this.

I haven't executed Octave tests: I don't bother with those unless I have made changes. If it passes on your side and on Travis I have a reasonable suspicion it should pass here. If it does not, that means that there is some funkiness in Octave & OS interoperability. For those things, I just think that hashing is just too fine-mazed to work reliably. The probability that an end user would be bothered by such an issue (e.g. rounding error of `100*epsilon`, 1% mismatch in figure location ...), is practically zero in my book.

**Comment by PeterPablo** [comment]
@egeerardyn, I marked your last three tests as unreliable. Please let's continue at #614 and afterwards #604, as soon as this PR is merged.

**Comment by egeerardyn** [comment]
R2014b in Arch Linux is pristine. 
Output in R2014a on Arch Linux:
```
The following tests are known to be unreliable. They, however, do not cause the build to fail.

           ACID(  8) peaks_contour            : hash 291479e4a695965b1166fdd083698188 != (4316434ef88b04b39c3fb88835461079)
           ACID( 10) peaks_contourf           : hash b1913e13f40beef74da65d6a463fef3a != (7b8c1c6618411a02b24e4d3a7fefe39d)
           ACID( 12) double_colorbar          : hash d5e1cdd066750729cc5cad0d22abaae1 != (8873cb127c10d0cb48ceb3c2bee6cee1)
           ACID( 17) colorbarLogplot          : hash 0f6344a9e3fc0b9cb38ea3918602b8ef != (585284ba0030d5a4dc08080402d9e772)
           ACID( 18) legendplot               : hash dafe7a9566de01abdee039ec8af9cd29 != (6fdecb3905efa3227de5869e202702be)
           ACID( 29) polarplot                : OK
           ACID( 30) roseplot                 : OK
           ACID( 31) compassplot              : OK
           ACID( 43) rlocusPlot               : OK
           ACID( 45) besselImage              : hash 2be5300f1f8dc4baf1b00931159408c5 != (85cb8eb6b29f9d05cc6491e06d198d4f)
           ACID( 47) zplanePlot1              : hash 9e26d9be3af2f058326e4602296594e0 != (934e422c29a4a2f4ec6f88ed838d9a0e)
           ACID( 48) zplanePlot2              : hash 0e83934b486d335b2a9d8d688279309c != (f9062776d7ef72383d91e75c232068d8)
           ACID( 49) freqResponsePlot         : hash c015d418587735998042396d8c93019d != (a6dea44359e9de74b487320f96aa5e7b)
           ACID( 56) spherePlot               : OK
           ACID( 57) surfPlot                 : hash b7d444599c2ca0345caa86ac5feb05bf != (e38457da9b8a459ee69c30c6f06a9128)
           ACID( 58) surfPlot2                : hash 79e4fb104ed1e63f8518b135f993c978 != (9ae67c1ca16a74102040ee2566553e7b)
           ACID( 60) meshPlot                 : OK
           ACID( 62) spectro                  : hash 6f983cb9a7627a7a67943a7f2a055b84 != (9e9c494a79f0e21433a0484b4db40fb6)
           ACID( 63) mixedBarLine             : OK
           ACID( 67) texrandom                : OK
           ACID( 69) latexmath2               : OK
           ACID( 70) parameterCurve3d         : OK
           ACID( 71) parameterSurf            : hash cd9fc11fdde062ad437ea87b95b4ce2c != (14b760693935cf31e831b01a8ecebd1d)
           ACID( 75) hist3d                   : OK
           ACID( 81) pColorPlot               : OK
           ACID( 82) hgTransformPlot          : OK
           ACID( 87) annotationAll            : hash cdab7295cba22a536c067f82e8780ae5 != (eb2926218c7b502b2ff63af7547073f6)
           ACID( 88) annotationSubplots       : hash 20c66d9564da246aefec3e646cf183a2 != (d6357170055983e54005cec158a6190c)
           ACID( 89) annotationText           : hash 657dd6d52f7b482ef0da830c0f7c9d5c != (bf69f61518aec9fb2c4fecff1d359f71)
           ACID( 90) annotationTextUnits      : hash 12ad28a8902e0555017e697f405f0287 != (5a874a814f93cd193ae2d521db22c106)
           ACID( 95) colorbarLabelTitle       : hash 3670447cbf70b47e2e8104ce5ebdf63e != (b25734f8cc15d031e46181535324b4c8)
           ACID( 96) textAlignment            : hash 035983dfbd573c6ed628285466ef663a != (dc2696d8b910e5ceb25b227ebd3667cc)

 19 of  32 tests failed. :-( 


Only the following tests determine the build outcome:
           ACID(  1) multiline_labels         : OK
           ACID(  2) plain_cos                : OK
           ACID(  3) sine_with_markers        : OK
           ACID(  4) markerSizes              : OK
           ACID(  5) markerSizes2             : OK
           ACID(  6) sine_with_annotation     : OK
           ACID(  7) linesWithOutliers        : OK
           ACID(  9) contourPenny             : OK
           ACID( 11) many_random_points       : OK
           ACID( 13) randomWithLines          : OK
           ACID( 14) double_axes              : OK
           ACID( 15) double_axes2             : OK
           ACID( 16) logplot                  : hash e14feacb3479ac5a8b4dd8294b096132 != (f5053634ccd961fc6d95c48df7f95be9)
           ACID( 19) legendplotBoxoff         : OK
           ACID( 20) moreLegends              : OK
           ACID( 21) zoom                     : OK
           ACID( 22) quiveroverlap            : OK
           ACID( 23) quiverplot               : OK
           ACID( 24) quiver3plot              : hash c92a354f492e093710abcf6687e0c1bd != (4cd7eb850e401213cc238dd9ba127686)
           ACID( 25) logicalImage             : OK
           ACID( 26) imagescplot              : OK
           ACID( 27) imagescplot2             : OK
           ACID( 28) stairsplot               : OK
           ACID( 32) stemplot                 : OK
           ACID( 33) stemplot2                : OK
           ACID( 34) bars                     : hash 7afb3f0415c45b489d97fdfd047692b1 != (4ec9811b3ba6e24bc10bddd655a56f40)
           ACID( 35) xAxisReversed            : OK
           ACID( 36) errorBars                : OK
           ACID( 37) errorBars2               : OK
           ACID( 38) subplot2x2b              : OK
           ACID( 39) manualAlignment          : OK
           ACID( 40) subplotCustom            : hash a40207079e1fd8116df7c027a21570b4 != (bf227900a56905bf7c39d653314d8667)
           ACID( 41) legendsubplots           : OK
           ACID( 42) bodeplots                : OK
           ACID( 44) mandrillImage            : OK
           ACID( 46) clownImage               : OK
           ACID( 50) axesLocation             : OK
           ACID( 51) axesColors               : OK
           ACID( 52) multipleAxes             : OK
           ACID( 53) scatterPlotRandom        : OK
           ACID( 54) scatterPlot              : OK
           ACID( 55) scatter3Plot             : OK
           ACID( 59) superkohle               : OK
           ACID( 61) ylabels                  : OK
           ACID( 64) decayingharmonic         : OK
           ACID( 65) texcolor                 : OK
           ACID( 66) textext                  : OK
           ACID( 68) latexInterpreter         : OK
           ACID( 72) fill3plot                : OK
           ACID( 73) rectanglePlot            : OK
           ACID( 74) herrorbarPlot            : OK
           ACID( 76) myBoxplot                : hash 5381722f000fae74074598ad0d602c51 != (a57e07a28a39cfd6dad154f2e714c3de)
           ACID( 77) areaPlot                 : OK
           ACID( 78) customLegend             : hash 41b082fc4d06edfe00730c37b3b5156e != (a5b8f598d5f811aeb16aa8716585b32a)
           ACID( 79) pixelLegend              : OK
           ACID( 80) croppedImage             : OK
           ACID( 83) scatterPlotMarkers       : OK
           ACID( 84) multiplePatches          : OK
           ACID( 85) logbaseline              : OK
           ACID( 86) alphaImage               : OK
           ACID( 91) imageOrientation_PNG     : OK
           ACID( 92) imageOrientation_inline  : OK
           ACID( 93) texInterpreter           : OK
           ACID( 94) stackedBarsWithOther     : OK
           ACID( 97) overlappingPlots         : OK
           ACID( 98) histogramPlot            : SKIPPED
           ACID( 99) alphaTest                : OK

  6 of  67 tests failed. :-(
```
Output files on https://gist.github.com/egeerardyn/b3131cc621ccfc78ff28

FYI, you can easily use the [`gist`](https://github.com/defunkt/gist) tool to upload those files :-)

**Comment by PeterPablo** [comment]
Those R2014a plots also fail on `develop` (see https://github.com/matlab2tikz/matlab2tikz/issues/615). So you are asking me to add those as `unreliable` as well?
(I am on the chat...)

**Comment by egeerardyn** [comment]
:+1:
